test/viewmodels/checkAnswers/sections/dispatch/DispatchAddressSummarySpec.scala

```diff
@@ -47,19 +46,12 @@ class DispatchAddressSummarySpec extends SpecBase with Matchers with UserAddress
           "when there's no answer for DispatchUseConsignorDetailsPage" in new Test(emptyUserAnswers) {
             DispatchAddressSummary.row() mustBe None
           }
```

Comment:
This test is irrelevant now

Reply:
It's relevant, but needs re-wording. It just shouldn't output if there's no address. Which, theoretically would never happen. Only if they jumped to the section. 

Thinking about it, it should probably output the `Not Provided` messages with a change link so they can fix it. Otherwise, they can't get out of that mess.